biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -213,6 +244,23 @@ subroutine disturbance_rates( site_in, bc_in)
     ! Calculate Disturbance Rates based on the mortality rates just calculated
     ! ---------------------------------------------------------------------------------------------
 
+    ! zero the diagnostic disturbance rate fields
+    site_in%potential_disturbance_rates(1:N_DIST_TYPES) = 0._r8
+
+    ! summarize how open the canopy is prior to resolving the disturbance
+    currentPatch => site_in%oldest_patch
```

Comment:
Has this changed on account of growth since we last calculated it?

Reply:
Since all patches age at the same rate, the order can only change due to fusion, right?